src/common/components/select-repositories-page/index.js

```diff
@@ -35,6 +36,7 @@ class SelectRepositoriesPage extends Component {
             <HeadingThree className={ styles.heading }>
               Add repos
             </HeadingThree>
+            <SelectRepositorySearch />
```

Comment:
I wouldn't add it here already in this PR, because if we merge just this we would end up with a component UI that doesn't have any functionality.

We can have all the bits in place, but add it to the page only if the functionality is finished.